src/main/java/uk/gov/companieshouse/api/accounts/CompanyAccountsApplication.java

```diff
@@ -58,6 +62,8 @@ public ObjectMapper objectMapper() {
 
     @Override
     public void addInterceptors(final InterceptorRegistry registry) {
+        registry.addInterceptor(loggingInterceptor)
+            .addPathPatterns("/transactions/{transactionId}/**");
```

Comment:
Shouldn't this interceptor be added to every route?

Reply:
The path pattern /transactions/{transactionId}/** includes all sub paths in company-accounts-api. The private path can be included when the private URIs are available.

Reply:
Can't we add it to every path so it doesn't need to be updated for the private URLs?